ckanext/xloader/plugin.py

```diff
@@ -8,6 +8,12 @@
 from . import action, auth, helpers as xloader_helpers, utils
 from .loader import fulltext_function_exists, get_write_engine
 
+IConfigDeclaration = None
+if toolkit.check_ckan_version("2.10"):
+    import yaml
+    import os
+    IConfigDeclaration = plugins.IConfigDeclaration
+
```

Comment:
Sorry, I mentioned it on the call but it took time before I finally managed to add a comment here.

You can simplify adding declarations using blanket:
```
try:
    config_declarations = tk.blanket.config_declarations
except AttributeError:
    config_declarations = lambda cls: cls

@config_declarations
class xloaderPlugin(plugins.SingletonPlugin):
...
```

As you see, here we define a no-op decorator if the blanket is unavailable so that it won't produce an error in 2.9. In this way you can completely remove the implementation of the interface from the class because it looks for `config_declaration.yaml` inside `ckanext/<plugin>` by default